### DISCUSSION TITLE
Error dalam menggunakan Fungsi LIME, walaupun nama dan jumlah prediktor antara data train di explainer dan data test di explain sama

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#644] [Q&A] [answered]
Melanjutkan pertanyaan sebelumnya #640, dan merefer ke jawaban di #486, Setelah dimasukkan fungsi Lime sbb ;

<img width="459" alt="image" src="https://user-images.githubusercontent.com/123436963/232988321-27e24473-27c5-4c39-a4d9-e580d8994e90.png">
 
didapatkan error sbb ;

<img width="429" alt="image" src="https://user-images.githubusercontent.com/123436963/232988509-90189812-7e80-474c-8ac4-523bf3301f00.png">

Nama dan jumlah prediktor antara data train di explainer dan data test di explain sudah sama tapi masih terjadi error , data tsb yaitu ;
1. Data train >> sc_train_tun_prop_lime

<img width="435" alt="image" src="https://user-images.githubusercontent.com/123436963/232988971-11320f9c-3d66-419e-8b16-449d2ab42243.png">

2. Data test >>  scot_dtest_clean_lime

<img width="422" alt="image" src="https://user-images.githubusercontent.com/123436963/232989174-ef8549a6-8c67-40c9-b03c-22991727283f.png">

Terhadap error yg masih terjadi tsb

<img width="429" alt="image" src="https://user-images.githubusercontent.com/123436963/232988509-90189812-7e80-474c-8ac4-523bf3301f00.png">

Apakah yang masih harus saya perbaiki dalam pengkodeannya ?

Terimakasih

**Comment by finesaaa** [comment]
Dear @Micoalgo, terima kasih atas pertanyaannya, akan segera dibantu oleh tim kami ya pak. Mohon ditunggu

**Comment by Micoalgo** [comment]
Ok Mba, Terimakasih

**Comment by hwulanayu** [comment, marked as answer]
Dear @Micoalgo,

Error tersebut menyebut bahwa hasil prediksi yang dikeluarkan model 'tuning_forest_hj3' harus berupa probability. Silakan cek output hasil prediksi apakah sudah berbentuk probability atau masih berbentuk kelas. Jika masih berbentuk kelas, coba diubah dulu type-nya.

**Comment by Micoalgo** [comment]
Ok, akan dicoba kembali. Terimakasih jawabannya
